pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py

```diff
@@ -67,6 +85,21 @@ def _get_values(attribute, text):
     values = regex.findall(text)
     return values
 
+
+def get_attr_value(attr_path: str) -> Optional[Any]:
+    try:
+        response = subprocess.check_output([
+            "nix",
```

Comment:
Maybe enable the experimental feature like is done further down below.

Reply:
Added.